.github/workflows/codeforafrica-deploy-review-app.yml

```
@@ -88,15 +88,12 @@ jobs:
 
   destroy_review_app:
     runs-on: ubuntu-latest
-    # only run when a pull request is closed
-    if: github.event.action == 'closed'
+    if: github.event_name == 'pull_request' && github.event.action == 'closed'
     steps:
-      - name: Destroy the review app
+      - name: Destroy review app
```

```suggestion
      - name: Destroy the review app
```
Don't see the need for this change

---

I will use this for testing purposes; removing the change.